extensions/amp-story/1.0/amp-story-desktop-panels.css

```diff
@@ -186,7 +185,3 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
   min-width: 320px !important;
   min-height: 533px !important;
 }
-
-.i-amphtml-story-desktop-panels > amp-story-page {
-  box-shadow: 0 50px 200px rgba(0, 0, 0, 0.7) !important
```

Comment:
Do we want to enforce this is `none`?

Reply:
Done, but to clarify this, having a `box-shadow` is fine. The value here is super high (200px!), making it overlap other pages that have an opacity themselves. They overlap each other, animating over several layers of opacity... The browser had to render that every 16ms :(